src/Microsoft.AspNet.Mvc.Core/ViewComponents/DefaultViewComponentHelper.cs

```diff
@@ -119,10 +128,18 @@ private ViewComponentDescriptor SelectComponent(Type componentType)
             {
                 if (descriptor.Type == componentType)
                 {
+                    _logger.LogError(
+                        "The view component of type '{ViewComponentTypeName}' was found successfully.",
+                        componentType.FullName);
+
                     return descriptor;
                 }
             }
 
+            _logger.LogError(
+                "The view component of type '{ViewComponentTypeName}' was not found.", 
+                componentType.FullName);
+
```

Comment:
I'm not convinced that this logging adds value. Can we split this out from the view-search logging (which we absolutely need)?